README.md

```diff
@@ -6,7 +6,8 @@
 
 This repository contains a set of tools to aid in the process of building, testing and releasing
 (open)SUSE based distributions and their corresponding maintenance updates. You can find more
-information in the [docs/processes.md](docs/processes.md).
+information in the [docs/processes.md](docs/processes.md). The [CONTENTS.md](CONTENTS.md) file
```

Comment:
This should be "in the docs/processes.md file" or "in docs/processes". Now it looks weird. I know it was wrong before already, but let's fix it.